Cauli/Florets/MapRemote/MapRemoteFloret.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  Copyright (c) 2018 cauli.works
```

Comment:
Would we still use 2018 or the current year 🙃?

Reply:
I am kind of tending towards just removing it completely. I honestly thing it's ineffective. There is no such thing as a legal instance "cauli.works" that could claim the copyright. Even if, it's open source. There are different people involved. 

Should we remove the copyright claim header on all files?

Reply:
Yep! Let's make this happen.
https://github.com/cauliframework/cauli/issues/224